iotilecore/setup.py

```diff
@@ -64,7 +64,10 @@
             'tile_based = iotile.core.hw.virtual.tile_based_device:TileBasedVirtualDevice'
         ],
         'iotile.recipe_action': [
-            'FlashBoardStep = iotile.core.hw.debug.flash_board_step:FlashBoardStep'
+            'FlashBoardStep = iotile.core.hw.debug.flash_board_step:FlashBoardStep',
```

Comment:
Is there a way these can all live in `archbuild` along with the other steps designed for internal manufacturing?  We ship `iotile-core` very broadly and I assume we already use archbuild other places in the manufacturing flow so it doesn't seem like it would be adding a new dependency.

Alternatively I would move it into `iotile-ship` so we can avoid including this code with every product that uses `iotile-core`:
https://code.eng.archsys.io/github.com/iotile/coretools/-/tree/iotileship/iotile/ship/actions

Reply:
I agree... I can move it into iotile-ship. I think that it would be better off there or archbuild...